hikari/impl/rest.py

```diff
@@ -2567,13 +2604,183 @@ async def _create_guild_channel(
             permission_overwrites,
             conversion=self._entity_factory.serialize_permission_overwrite,
         )
+        body.put("default_auto_archive_duration", default_auto_archive_duration, conversion=time.timespan_to_int)
 
         response = await self._request(route, json=body, reason=reason)
         assert isinstance(response, dict)
         channel = self._entity_factory.deserialize_channel(response)
         assert isinstance(channel, channels_.GuildChannel)
         return channel
 
+    async def create_message_thread(
```

Comment:
```suggestion
    async def create_thread_from_message(
```
?

Reply:
Makes more sense to me, but since Discord calls them message threads, ignore this

Reply:
they are message threads though, they share the ID of the message they were created on and a message can only have one